src/multiconfigwidget.h

```diff
@@ -0,0 +1,38 @@
+#ifndef MULTICONFIGWIDGET_H
+#define MULTICONFIGWIDGET_H
+
+#include <KConfigGroup>
+#include <functional>
+#include <QComboBox>
```

Comment:
forward declare

Reply:
I cant forward declare KConfigGroup but I did it with QPushButton and QComboBox